libs/common-services/src/main/java/uk/gov/di/ipv/core/library/helpers/VerifiableCredentialParser.java

```diff
@@ -24,7 +24,9 @@ public class VerifiableCredentialParser {
     private static final Logger LOGGER = LogManager.getLogger();
     private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
     private static final ObjectMapper LAX_OBJECT_MAPPER =
-            new ObjectMapper().configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+            new ObjectMapper()
+                    .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
+                    .configure(DeserializationFeature.READ_UNKNOWN_ENUM_VALUES_AS_NULL, true);
```

Comment:
The only known place where we have unknown enum values is in one of our contract tests (@DanCorderIPV is following up with Lime on this), but it probably makes sense to be forgiving on this anyway.